RecoTracker/TkHitPairs/src/HitPairGeneratorFromLayerPair.cc

```diff
@@ -102,10 +101,26 @@ HitDoublets HitPairGeneratorFromLayerPair::doublets( const TrackingRegion& regio
 
   const RecHitsSortedInPhi& outerHitsMap = theLayerCache(outerLayerObj, region, iEvent, iSetup);
   if (outerHitsMap.empty()) return HitDoublets(innerHitsMap,outerHitsMap);
-
   HitDoublets result(innerHitsMap,outerHitsMap); result.reserve(std::max(innerHitsMap.size(),outerHitsMap.size()));
+  doublets2(region,
```

Comment:
was this supposed to be "doublets"? I don't see a definition of doublets2 in this PR